docs/reference/migration/migrate_7_0/settings.asciidoc

```diff
@@ -40,3 +40,16 @@ will be removed in the future, thus requiring HTTP to always be enabled.
 This setting has been removed, as disabling http pipelining support on the server
 provided little value. The setting `http.pipelining.max_events` can still be used to
 limit the number of pipelined requests in-flight.
+
+==== Cross-cluster search settings renamed
+
+The cross-cluster search remote cluster connection infrastructure is also used
+in cross-cluster replication. This means that the setting names
+`search.remote.*` used for configuring cross-cluster search belie the fact that
```

Comment:
belie?

Reply:
It's one of my favorite words, my "lodestar" or "malign" if you will. 😇 

I use it to mean here that the names `search.remote.*` do not reveal the fact that these settings also apply to cross-cluster replication, or connections to remote clusters in general.

Reply:
ok :) I did not know that